packages/react-storage/src/components/StorageBrowser/__tests__/createStorageBrowser.spec.tsx

```diff
@@ -12,9 +13,30 @@ const config = {
 };
 
 describe('createStorageBrowser', () => {
-  it('returns a StorageBrowser', () => {
+  it('returns a StorageBrowser', async () => {
     const { StorageBrowser } = createStorageBrowser({ config });
 
-    expect(render(<StorageBrowser />).container).toBeDefined();
+    await waitFor(() => {
+      expect(render(<StorageBrowser />).container).toBeDefined();
+    });
```

Comment:
Added this because there was a console.error being logged during tests about testing components that updated state in `act`. Saw this post: https://stackoverflow.com/a/68331605/6995752 and saw that `waitFor` was being used elsewhere so I added it here